sdk/servicebus/service-bus/test/stress/stressTestsBase.ts

```diff
@@ -21,8 +21,24 @@ import {
   SnapshotOptions,
   TrackedMessageIdsInfo
 } from "./utils";
+import * as appInsights from "applicationinsights";
 const writeFile = util.promisify(fs.writeFile);
-const appendFile = util.promisify(fs.appendFile);
+const nodeAppendFile = util.promisify(fs.appendFile);
```

Comment:
But you're not using this anywhere.

Reply:
You're right! (for a short period of time I was doing "both" but then decided to just remove it altogether). I'll kill it.

Reply:
Just to mention, there's some more to remove/change (for instance, we're apparently still using writeFile).

Reply:
(near-future PR)

Reply:
Yes, I saw that!